src/components/StatelessTransaction.js

```diff
@@ -15,6 +15,16 @@ const StatelessTransaction = props => {
   const { setUserApproval, userApproval } = props
   const { canSend, handleSubmit } = props
 
+  // setState doesn't seem to work in SetProxy/handleSubmit, so just
+  // modifying the DOM manually here. (https://imgur.com/a/i0Qsyq1)
```

Comment:
Linking to an image (and an indirect link at that) makes me feel weird. Is it possible to very briefly describe that error instead?  
Also, unsure, but maybe this needs a `//TODO investigate`.